### DISCUSSION TITLE
PV functions with same arguments.

### DISCUSSION THREAD

**Discussion by sunnyasquabla** [FeynCalc/feyncalc#263] [Q&A] [answered]
Hi,
I want to make all the PV functions appearing in the expression below to have same arguments
like for example,
C0(m1,m2,m3, k1,k2,k3)
C1(m1,m2,m3, k1,k2,k3)

all having same arguments (m1,m2,m3, k1,k2,k3)
![image](https://github.com/FeynCalc/feyncalc/assets/107980747/b5190fb5-7af3-441c-ae67-b6bf5ad8d987)

Is there any way to do this?
Thanks.

**Comment by vsht** [comment, marked as answer]
Hi,

have a look at PaVeOrder. The manual has couple of examples for that.

Am 16.04.24 um 10:01 schrieb Sunny:

**Comment by sunnyasquabla** [comment]
It worked,
Thank You Sir.
